src/OpenLoco/src/Windows/Main.cpp

```diff
@@ -72,12 +72,15 @@ namespace OpenLoco::Ui::Windows::Main
     // 0x00468FFE
     void hideGridlines()
     {
-        auto window = WindowManager::getMainWindow();
-        if (window == nullptr || !window->viewports[0]->hasFlags(ViewportFlags::gridlines_on_landscape))
-            return;
+        if (!Config::get().hasFlags(Config::Flags::gridlinesOnLandscape))
+        {
```

Comment:
I'd prefer an early return over wrapping the code in an if block tbh. What do you think? It's not a big deal either way.